.ci.yaml

```diff
@@ -155,6 +155,17 @@ targets:
       - cipd_packages/ruby/**
       - .ci.yaml
 
+  - name: Linux ktlint
```

Comment:
Do you need one for other platforms?

Reply:
No, I only plan to use this for one of the linux checks